SYCL/USM/copy.cpp

```diff
@@ -27,12 +27,11 @@ struct test_struct {
   long long d;
   half e;
   float f;
-  double g;
```

Comment:
Could we have another struct with the double and give it the same treatment as just double?

Reply:
I think this can be addressed post-commit in another PR to fix the currently failing tests in CI asap.

@intel/llvm-gatekeepers would you agree with that? If so, please merge it in.